website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,207 @@
+
+This will ensure consistency when you are working with Podman from all your different environments
+
+- Set an alias to use the remote Podman without modifying how you start the Podman CLI:
+
+  ```shell-session
+  $ alias podman='podman --remote --uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock"'
+  ```
+
+  <details>
+
+  <summary>
+  On a custom Podman Machine, the remote Podman Machine destination might be different. Find it:
+  </summary>
+  <div>
+
+  A custom Podman machine might have a different name.
+  You need to find your Podman machine remote destination.
+
+  1. Start a session in your WSL distribution:
+
+     ```shell-session
+     > wsl
+     ```
+
+  1. The Podman machine shares sockets in a `/mnt/wsl/podman-sockets/` subdirectory named after the Podman machine name.
+     List the available sockets, and identify your remote Podman machine connection in the list.
+
+     ```shell-session
+     $ ls --numeric--uid-gid --recursive /mnt/wsl/podman-sockets/
+     ```
+
+  1. To define the Podman machine remote destination, prepend the found path with `uri://`:
+
+     For the default Podman machine:
+
+     - Rootful Podman: `unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock`
+     - Rootless Podman: `unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-user.sock`
+
+  </div>
+  </details>
+
+  <details>
+  <summary>
+  On a custom WSL distribution with a Podman v4 client, consider configuring Podman directly, rather than your terminal session:
+  </summary>
+
+  With Podman v4, you can use the Podman configuration file.
+  Podman does not need the `--remote` flag when you have set a remote service destination.
+
+  To assign the remote service destination, in your WSL distribution, configure the `[engine.service_destinations]` table in the Podman configuration file.
+
+  1. Create the Podman configuration file if it does not exist.
+
+     ```shell-session
+     $ mkdir -p ~/.config/containers/containers.conf.d
+     $ touch ~/.config/containers/containers.conf.d/destinations.conf
+     ```
+
+  1. Add these lines in the `~/.config/containers/containers.conf.d/destinations.conf` file:
+
+     - Default Rootful Podman machine:
+
+       ```toml
+       [engine.service_destinations]
+       [engine.service_destinations.production]
+       uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock"
+       ```
+
+     - Default Rootless Podman machine:
+
+       ```toml
+       [engine.service_destinations]
+       [engine.service_destinations.production]
+       uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-user.sock"
+       ```
+
+  </details>
+
+## Testing the connection to the remote Podman machine
+
+On your WSL distribution, the Podman CLI communicates with your Podman machine.
```

Comment:
don't you want to make the documentation more lively with a screenshot?

Reply:
Added CLI outpu samples.